package.json

```diff
@@ -34,7 +34,6 @@
     "node-sass": "^4.14.1",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "postcss-loader": "^3.0.0",
-    "remove-files-webpack-plugin": "^1.1.3",
```

Comment:
Instead of removing, I suggest to uninstall the `remove-files-webpack-plugin` package and then you'll have the package-lock.json updated with the correct dependencies.

Reply:
This file has an inconsistency against the `package-lock.json`

Reply:
As discussed on Slack this turned out to be not an issue.